device-types/Datacom/DM4270-48XS-plus-6CX.yaml

```diff
@@ -0,0 +1,147 @@
+---
+manufacturer: Datacom
+model: DM4270 48XS+6CX
+slug: datacom-dm4270-48xs-plus-6cx
+part_number: 800.5304.xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: L2/L3/MPLS switch, wire speed, with 48 10 GE optical ports on SFP+ connectors and 6 optical ports multi-rate 100GE/40GE on QSFP28 connectors.
+  1U metal cabinet for 19” rack with 4 ventilation modules. MPLS license and power supplies and optical modules must be purchased separately
+weight: 6.28
+weight_unit: kg
+console-ports:
+  - name: Console0
+    type: rj-45
+  - name: Console1
+    type: usb-a
+console-server-ports:
+  - name: Storage
+    type: usb-b
```

Comment:
Is this actually used as a console server port? Or is it just for the switch to access a USB storage device? If the latter, please remove this definition as it would be out of scope.

Reply:
removed